android/KMEA/app/src/main/java/com/tavultesoft/kmea/ModelPickerActivity.java

```diff
@@ -275,7 +275,7 @@ public void onBackPressed() {
 
   // Uses the repo dataset's master lexical model list to create a filtered adapter for use here.
   // As this one is specific to this class, we can implement Activity-specific functionality within it.
-  static private class FilteredLexicalModelAdapter extends NestedAdapter<LexicalModel, Dataset.LexicalModels, String> {
+  static public class FilteredLexicalModelAdapter extends NestedAdapter<LexicalModel, Dataset.LexicalModels, String> {
```

Comment:
Why does this now need to become public? Maybe we can have a call to walk me through this.

Reply:
This Adapter is used for 2 screens now:
[Model Picker](https://docs.google.com/document/d/1LQW6F1GpqmMxOXm-tEvranUu_T6ck_M0Llk2fyJmqcI/edit#bookmark=id.wdyzeog744g4)
and FV Keyboard settings page.

Reply:
We can revert this to private in the future when the Keyman app is refactored to use the same "keyboard settings" menu as FV